src/homework10/SecondTask.java

```diff
@@ -0,0 +1,27 @@
+package homework10;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class SecondTask {
```

Comment:
Pls, test your code for example with data
        numbers.add(9);
        numbers.add(50000);
        numbers.add(950);
        numbers.add(0);
        numbers.add(155);
The problem that after removing value your list will change(move elements to the left). As a result some values skip check if (numbers.get(i) > X) {

Reply:
fixed